packages/loot-core/bin/copy-migrations

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -e
+#!/bin/bash -e
```

Comment:
Better yet, this should be `/usr/bin/env bash -e`

Reply:
Merging this for now to fix the immediate issue but I think this may be worth exploring more. It seems like some OSes don’t allow passing arguments when using `env` so that may not work here. If you find a version that’s more portable (and an OS where the current system breaks) feel free to open a PR!

Reply:
The way I do it is something like

```sh
#!/usr/bin/env bash

set -e
```

However, I just wanted to pipe in with my opinion. Its not a big problem at all!

Reply:
missed the discussion on this, sorry - agreed, env makes the most sense - perhaps also related to someone exploring a unification of bash scripts is also the classic 'set -euxo pipefail' preamble to make working with bash scripts easier (https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425)

btw. thanks a lot to all of you working on actual, it's an absolute godsent to have an opensource YNAB style budgeting toolkit :) i'll try to contribute more in the future!